doc/manual.rst

```diff
@@ -989,10 +989,12 @@ Ordinal types have the following characteristics:
 - Ordinal types are countable and ordered. This property allows the operation
   of functions such as `inc`, `ord`, and `dec` on ordinal types to
   be defined.
-- Ordinal values have the smallest possible value. Trying to count further
-  down than the smallest value produces a panic or a static error.
-- Ordinal values have the largest possible value. Trying to count further
-  than the largest value produces a panic or a static error.
+- Ordinal values have the smallest possible value, accessible with `low(type)`.
+  Trying to count further down than the smallest value produces a panic or
+  a static error.
+- Ordinal values have the largest possible value, accessible with `high(type)`.
```

Comment:
```suggestion
- Ordinal values have a largest possible value, accessible with `high(type)`.
```

Reply:
docs, like code, should be DRY, just say:
```
likewise with `high(type)` for the largest possible value.
```